src/OpenTelemetry.Exporter.Zipkin/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+* Zipkin will now write numeric, boolean, and floating-point tag values as
```

Comment:
What is the impact here?

Reply:
I was thinking this was a bug fix to make it compliant with the spec. But checking the spec it says for some reason long/double/bool should be converted to string using en-US. Seems odd, but I reverted.